Lib/test/test_typing.py

```diff
@@ -7258,7 +7237,7 @@ def test_all(self):
         # Context managers.
         self.assertIn('ContextManager', a)
         self.assertIn('AsyncContextManager', a)
-        # Check that io and re are not exported.
+        # Check that former namespaces io and re are not exported.
```

Comment:
We can surely just delete this line and the next two lines? It seems like a redundant test now the namespaces are being removed entirely.